cmd/broker/ingress/main.go

```diff
@@ -63,7 +62,7 @@ func main() {
 		ingress.Port(env.Port),
 		ingress.ProjectID(projectID),
 		metrics.PodName(env.PodName),
-		metrics.ContainerName(containerName),
+		metrics.ContainerName(component),
```

Comment:
I'd add a comment of why we are doing this...

Reply:
I just made this consistent with what the fanout & retry files do, I think ContainerName was a way to include "ingress" as the container name while keeping the metrics working with the "broker" component, but now we can specify the metric name via this PR